admin/qtx_configuration.php

```diff
@@ -1392,7 +1392,7 @@ function qtranxf_conf() {
 					</fieldset><br />
 					<textarea id="highlight_mode_custom_css" name="highlight_mode_custom_css" style="width:100%"><?php echo esc_attr($highlight_mode_custom_css); ?></textarea>
 					<br />
-					<small><?php echo __('To reset to default, clear the text.', 'qtranslate').' '; printf(__('The color in use is taken from your profile option %s, the third color.', 'qtranslate'), '"<a href="'.admin_url('/profile.php').'">'.__('Admin Color Scheme').'</a>"') ?></small>
+					<small><?php echo __('To reset to default, clear the text.', 'qtranslate').' '; printf(__('The color in use is taken from your profile option %s, the third color.', 'qtranslate'), '"<a href="'.admin_url('/profile.php').'">'.__('Admin Color Scheme', 'qtranslate').'</a>"') ?></small>
```

Comment:
As I said above, I've spotted that already existing translation because I was lucky.
I suggest that if you want to use a specific title of something already existing, like this **Admin Color Scheme**, maybe you can add a comment to it, strings like Plugin titles, trademarks or author names can sometimes be confused with strings to translate. In this particular case a comment to explain it's context is the best way to inform translators what to do with that particular string.